cray_product_catalog/catalog_update.py

```diff
@@ -271,7 +292,12 @@ def main():
     if VALIDATE_SCHEMA:
         validate_schema(data)
 
-    update_config_map(data, CONFIG_MAP, CONFIG_MAP_NAMESPACE)
+    LOGGER.debug("Splitting cray-product-catalog data")
+    main_cm_data, prod_cm_data = split_catalog_data(data)
+
+    update_config_map(main_cm_data, CONFIG_MAP, CONFIG_MAP_NAMESPACE)
+    if prod_cm_data:
```

Comment:
@lathanm not sure a empty dictionary will be treated as false. If tested cool, else you have a small helper function to check 
```
def is_product_cm_exist(prod_cm_data: Dict) -> bool:
    """ Function to check if product cm exists or not """
    if prod_cm_data is None or prod_cm_data == {}:
        return True
   else:
       return False
```

What @paulose-kuriakose /@anoop/@abhishek suggest ?

Reply:
`if prod_cm_data:` : It holds good since all empty objects are False in python.